main.go

```diff
@@ -1,12 +1,31 @@
 package main
 
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/hisshoes/crypto-rebalancer/pkg/http/rest"
+	"github.com/hisshoes/crypto-rebalancer/pkg/portfolio"
+	"github.com/hisshoes/crypto-rebalancer/pkg/storage/memory"
+)
+
 func main() {
 	dbType := "memory"
 
+	var p portfolio.Service
+
 	switch dbType {
 	case "memory":
+		s := new(memory.Storage)
+		p = portfolio.NewService(s)
 
 	default:
 		panic("Unknown database")
 	}
+
+	router := rest.Handler(p)
+
+	fmt.Println("Portfolio server starting")
```

Comment:
As below, I'd stick to using the `log` package. I recommend https://github.com/sirupsen/logrus but it might be overboard for right now. It has a nice way of including metadata on log messages (such as parameter values etc).